src/_includes/components/footer.njk

```diff
@@ -45,10 +48,25 @@
             },
             body
           }).then((res) => {
-            form.reset();
-            success.classList.remove("hidden");
+            if (res.status === 400) throw Error("The email you provided was invalid");
+           showSuccessMsg()
+          }).catch((err) => {
+            showErrorMsg()
           })
         });
+
+        function showSuccessMsg() {
+          form.reset();
+          //NOTE: Maybe add a 'green' success style
+          success.innerHTML = `Thanks for signing up`
```

Comment:
Can use double quotes here instead of backticks

Reply:
Fixed!